peer/brontide.go

```diff
@@ -4111,6 +4118,29 @@ func (p *Brontide) handleRemovePendingChannel(req *newChannelMsg) {
 	p.addedChannels.Delete(chanID)
 }
 
+// handlePeerStorageMessage handles `PeerStorage` message, it stores the message
+// and sends it back to the peer as an ack.
+func (p *Brontide) handlePeerStorageMessage(msg *lnwire.PeerStorage) error {
+	// Check if we have the feature to store peer backup enabled.
+	if p.cfg.PeerDataStore != nil {
```

Comment:
should we send an error/warning if our peer is sending us this message but we have not advertised the bit?

Reply:
Send a warning and disconnect?

Reply:
I'm inclined to agree with @ellemouton here.